### ISSUE TITLE
twisted.web.http_headers.Headers capitalizes some headers incorrectly

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5054]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| ivank reported|
|-|-|
|Trac ID|trac#5054|
|Type|defect|
|Created|2011-04-18 03:06:05Z|
|Branch|https://github.com/twisted/twisted/tree/capitalize-headers-5054|

There are a few headers besides `WWW-Authenticate` that need to be in `_caseMappings`.

(Patch incoming. This should be pretty simple.)

Attachments:

* [5054-00-capitalize-headers-correctly.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/938/9382da7d06dd452f6e5756535593956669b1ac7d/249a670854527030c59e26a6cca5928606d3ee57.patch) (2331 bytes) - added by ivank on 2011-04-18 03:16:29Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5054 5054
type__defect defect
reporter__ivank ivank
priority__normal normal
milestone__ 
branch__branches_capitalize_headers_5054 branches/capitalize-headers-5054
branch_author__therve therve
status__closed closed
resolution__fixed fixed
component__web web
keywords__easy easy
time__1303095965000000 1303095965000000
changetime__1303300713000000 1303300713000000
version__None None
owner__therve therve
cc__ivank cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve set owner to @therve|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve commented|
|-|-|

(In [31706]) Branching to 'capitalize-headers-5054'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_6"></a>|@therve commented|
|-|-|

(In [31707]) Apply patch

Refs [#5054](https://github.com/twisted/twisted/issues/5054)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_7"></a>|@therve commented|
|-|-|

Looks good!

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_8"></a>|@therve set status to `closed`|
|-|-|

(In [31708]) Merge capitalize-headers-5054

Author: ivank
Reviewer: therve
Fixes: [#5054](https://github.com/twisted/twisted/issues/5054)

Correctly capitalize specific headers in twisted.web.http_headers.Headers.
